packages/ds-medicare-gov/src/styles/base/_override.typography.scss

```diff
@@ -28,26 +26,18 @@
   }
 }
 
-.ds-title,
-.ds-text-heading--4xl {
-  font-weight: 300;
-}
-
 .ds-h1,
 .ds-text-heading--3xl {
-  font-size: $font-size-2xl;
   line-height: $spacer-5;
   margin-bottom: 44px;
   @media screen and (min-width: $width-md) {
-    font-size: $font-size-3xl;
```

Comment:
I'm tracking down the sources of the loki differences, and this is the source of one of them. The font-size used to be different in medium screens vs everything smaller than that. This is responsive difference is not something we've tried to capture with tokens before. @babsdenney, is this mobile/desktop difference correct? Or should we normalize it to one size? It's too bad that the size is in pixels.

Reply:
For my own notes: On the main branch, it's 32px on small screens and 40px on large screens. On this branch, it's 30px on small screens and 40px on large screens.

Reply:
Theoretically if we wanted to capture different values for different screen sizes in the tokens, we could use this pattern a pattern like `typography-heading-3xl__font-size--width-sm` 🤷‍♂️